src/string-view.h

```diff
@@ -187,6 +187,59 @@ inline bool operator>=(string_view x, string_view y) noexcept {
   return x.compare(y) >= 0;
 }
 
+// non-member append.
+inline std::string& operator+=(std::string& x, string_view y) {
+  x.append(y.data(), y.size());
```

Comment:
maybe call the param `lhs` and `rhs`?  I don't know what the convention are here... just a thought.

Reply:
I was actually going to use exactly those names, until I saw the functions above used `x` and `y`.